<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sMax</class>
 <widget class="QDialog" name="sMax">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="cameraLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cameraColorLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1341</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="faceDetectionButton">
      <property name="text">
       <string>Detect Face And Eye</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="objectDetectionButton">
      <property name="text">
       <string>Detect Object</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="colorDetectionButton">
      <property name="text">
       <string>Detect Color</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>570</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="hMinLabel">
        <property name="text">
         <string>Hmin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hMin">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="valueHMinLabel">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>630</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="sMinLabel">
        <property name="text">
         <string>Smin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sMin">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="valueSMinLabel">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>690</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="vMinLabel">
        <property name="text">
         <string>Vmin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="vMin">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="valueVMinLabel">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>570</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="hMaxLabel">
        <property name="text">
         <string>Hmax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hMax">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="value">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="valueHMaxLabel">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>630</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="sMaxLabel">
        <property name="text">
         <string>Smax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sMax_2">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="valueSMaxLabel">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>690</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="vMaxLabel">
        <property name="text">
         <string>Vmax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="vMax">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="valueVMaxLabel">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
